src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -113,6 +114,7 @@ export default {
 				this.flssLoanSearch.gender = filter.gender;
 			} else if (filter.sortBy) {
 				this.flssLoanSearch.sortBy = filter.sortBy;
+				this.loanSearchState.sortBy = filter.sortBy;
```

Comment:
Why does this case need to update `loanSearchState` but not the others?

Reply:
the query uses `loanSearchState` for the initial query call on first mount